src/bindings/python/src/pyopenvino/graph/shape.cpp

```diff
@@ -26,11 +26,35 @@ void regclass_graph_Shape(py::module m) {
     shape.def(py::init<const ov::Shape&>(), py::arg("axis_lengths"));
     shape.def(py::init<const std::string&>(), py::arg("shape"));
     shape.def(
-        "__eq__",
```

Comment:
You don't need to delete this method -- Shape is inherited from std::vector and correctly checks equality of elements.